XenAdmin/Commands/MoveVMCommand.cs

```diff
@@ -67,7 +67,7 @@ public MoveVMCommand(IMainWindow mainWindow, VM vm)
         protected override void ExecuteCore(SelectedItemCollection selection)
         {
 
-            if (selection.AllItemsAre<VM>() && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())
+            if (selection.Count > 1 && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())

```

Comment:
If I'm not mistaken on older servers we launch the MoveVMDialog which moves one VM at a time.